src/actions/gif.js

```diff
@@ -0,0 +1,14 @@
+const REQUEST_GIF = 'REQUEST_GIF';
+const LOADING_GIF = 'LOADING_GIF';
+const GET_GIF = 'GET_GIF';
+const SET_GIF_ERROR = 'SET_GIF_ERROR';
+
+const requestGIF = id => ({ type: REQUEST_GIF, payload: id });
```

Comment:
As a note, it is a good practice to wrap the payload inside an object (this convention was first created by the `Flux` team, and it's recommend to use also in `Redux` for consistency).

```js
payload: { gif },
```